docs/guides/pgpool/reconfigure/reconfigure-pgpool.md

```
@@ -186,15 +186,15 @@ spec:
 
 Here,
 
-- `spec.databaseRef.name` specifies that we are reconfiguring `pp-csutom` pgpool.
+- `spec.databaseRef.name` specifies that we are reconfiguring `pp-custom` pgpool.
 - `spec.type` specifies that we are performing `Reconfigure` on our pgpool.
 - `spec.configuration.configSecret.name` specifies the name of the new secret.
 - Have a look [here](/docs/guides/pgpool/concepts/opsrequest.md#spectimeout) on the respective sections to understand the `timeout` & `apply` fields.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)